app.plugin.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./plugin/build/withHeap');
```

Comment:
I couldn't figure out how to run the plugin. (`"plugins": [ "@heap/heap" ],`, `"plugins": [ "heap" ],`, etc.

I don't see this file in the zip when I look at the package so maybe the packaging process has changed or maybe I'm doing something else wrong:

```
$ npm pack
$ tar -ztvf heap-react-native-heap-0.21.0.tgz |grep plugin
-rw-r--r--  0 0      0         541 Oct 26  1985 package/dist/plugin/src/withHeap.js
```

Reply:
its `"plugins": ["@heap/react-native-heap"]` it should match what is in package.json but you are right we should add it to the doc

Reply:
Yes `app.plugin.js` needs to be there. that's what expo looks for to load the plugin

Reply:
Probably why you got that undefined from the expo install